src/index.js

```diff
@@ -1,79 +1,67 @@
-        localStorage.setItem('todos', JSON.stringify(todoList));
-        window.location.reload();
-      });
-      itemContent.classList.add('p-todo-desck');
-      itemContent.innerHTML = todo.description;
-      myContentWrapper.appendChild(checkBox);
-      myContentWrapper.appendChild(itemContent);
-      listItem.appendChild(myContentWrapper);
-      listItem.appendChild(moreIcon);
-      myWrapper.append(listItem);
+  const todos = JSON.parse(localStorage.getItem('todos'))
+    ? JSON.parse(localStorage.getItem('todos'))
+    : [];
+  todos.forEach((todo) => {
+    const listItem = document.createElement('li');
+    listItem.classList.add('list-item');
+    const itemContent = document.createElement('input');
+    const checkBox = document.createElement('input');
+    const moreIcon = document.createElement('img');
+    const myContentWrapper = document.createElement('div');
+    myContentWrapper.classList.add('ct-wrapper');
+    moreIcon.classList.add('icon-cust');
+    moreIcon.setAttribute('src', `${deleteImage}`);
+    moreIcon.setAttribute('width', '20');
+    moreIcon.setAttribute('heigt', '20');
+    moreIcon.addEventListener('click', () => {
+      deleteTodo(todo.index);
     });
-  } else {
-    todos.forEach((todo) => {
-      const listItem = document.createElement('li');
-      listItem.classList.add('list-item');
-      const itemContent = document.createElement('p');
-      const checkBox = document.createElement('input');
-      const moreIcon = document.createElement('img');
-      const myContentWrapper = document.createElement('div');
-      myContentWrapper.classList.add('ct-wrapper');
-      moreIcon.classList.add('icon-cust');
-      moreIcon.setAttribute('src', `${moreImage}`);
-      moreIcon.setAttribute('width', '6');
-      moreIcon.setAttribute('heigt', '10');
-      checkBox.setAttribute('type', 'checkbox');
-      checkBox.checked = todo.isComplete;
-      checkBox.addEventListener('change', () => {
-        const isComplete = checkTodo(todo.isComplete);
-        todo.isComplete = isComplete;
-        localStorage.setItem('todos', JSON.stringify(todos));
-        window.location.reload();
-      });
-      itemContent.classList.add('p-todo-desck');
-      itemContent.innerHTML = todo.description;
-      myContentWrapper.appendChild(checkBox);
-      myContentWrapper.appendChild(itemContent);
-      listItem.appendChild(myContentWrapper);
-      listItem.appendChild(moreIcon);
-      myWrapper.append(listItem);
+    checkBox.setAttribute('type', 'checkbox');
+    checkBox.checked = todo.completed;
+    itemContent.style.textDecoration = todo.completed ? 'line-through' : 'none';
+    itemContent.classList.add('todoinput');
+    itemContent.disabled = true;
+    listItem.addEventListener('click', () => {
+      itemContent.disabled = false;
     });
-  }
+    itemContent.addEventListener('keyup', (e) => {
+      if (e.key === 'Enter') {
+        updateDescription(todo.index, e.target.value);
+      }
+    });
```

Comment:
- [OPTIONAL] Again, for an awesome User Experience, you can also add `focusout` eventListener. Why? A user might not know they are expected to press ENTER for them to update a task's description. Consider doing something like this:

```JavaScript
itemContent.addEventListener('focusout', (e) => {
  updateDescription(todo.index, e.target.value);
});
```

`foucusout` event listener will update the description once the user clicks out the input field.